keyboards/dztech/dz60rgb/keymaps/mekanist/keymap.c

```diff
@@ -50,7 +49,6 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
 
 
 
-extern led_config_t g_led_config;
 void rgb_matrix_layer_helper (uint8_t red, uint8_t green, uint8_t blue, bool default_layer) {
```

Comment:
/cries

Didn't catch this before..... should not have default layer, any more.